mkdocs.yml

```diff
@@ -12,19 +12,20 @@ nav:
     - Timings: Timings.md
     - FAQ: Frequently-asked-questions-(FAQ).md
     - For developers: Developers.md
-  - Tutorial:
+  - Tutorials:
     - Ribosomes on ER microsomes: tutorials/Tutorial.md
   - Discussions: https://github.com/SBC-Utrecht/pytom-match-pick/discussions
   - Report a bug: https://github.com/SBC-Utrecht/pytom-match-pick/issues
 
 theme:
+  logo: images/60S_masked.png
   name: material
   features:
     - navigation.instant
     # - navigation.instant.progress  # possible progress bar, but site is too small to justify
     - navigation.tracking
     - navigation.tabs
-    - navigation.tabs.sticky
+    # - navigation.tabs.sticky
```

Comment:
Just for my information: Why did you decide against sticky tabs?

Reply:
Good that you ask. I though it kept more info on screen on mobile, but I just double checked and mobile already has a separate interface. So i will switch back to sticky tabs

Reply:
```suggestion
    - navigation.tabs.sticky
```